Libraries/VaahStripe.md

```diff
@@ -76,12 +76,100 @@ Reference url: https://stripe.com/docs/api
         'state'         => 'xxxxxx'                     // optional
     ];
     
-    $return_url    // URL to redirect your customer back to after they authenticate or cancel their payment
+    $return_url = http://localhost/vaahcms/public/      // URL to redirect your customer back to after they authenticate or cancel their payment
```

Comment:
@wri-vikram-wri231 you should use problem syntax, I have seen similar errors in previous PRs as well.

It should be:
```
$return_url = "http://localhost/vaahcms/public/";
```